src/api/queues.js

```diff
@@ -113,43 +115,29 @@ router.patch(
   [
     requireCourseStaffForQueue,
     requireQueue,
-    check('name').isLength({ min: 1 }),
+    check('name')
+      .optional({ nullable: true })
```

Comment:
The more I think about this, I'm not sure if we want `nullable` or not - I think this would theoretically allow you to set the name of a queue to `null`, which we don't want. Can you look into this and maybe add some tests to assert that we can't accidentally do that? Same for `open` and `location` 🙂